tools/tojs/anysdk-common.ini

```diff
@@ -11,7 +11,7 @@ android_headers = -I%(androidndkdir)s/platforms/android-14/arch-arm/usr/include
 android_flags = -D_SIZE_T_DEFINED_
 
 clang_headers = -I%(clangllvmdir)s/%(clang_include)s
-clang_flags = -nostdinc -x c++ -std=c++17 -fsigned-char -U__SSE__ -U__ARM_NEON
+clang_flags = -nostdinc -x c++ -std=c++17 -fsigned-char -mfloat-abi=soft -U__SSE__
```

Comment:
any reason to change the flags?

Reply:
By specifying float-abi we can support the NEON header usages. Although not particularly helpful for JSB binding generation, this flag is needed anyways when compiling for android platforms. So I think it would be a better practice to unify the two compiling flags.